config/origin_defaults.py

```diff
@@ -0,0 +1,3 @@
+ORIGIN_SIGNING_KEY = ("0x1fc2b755568ce8402e422f8fd0da54d3"
+                      "84f42962c8f925116964f39245d429e0")
+TEST_DATABASE_URI = 'postgresql://localhost/unittest'
```

Comment:
We can remove this safely as we are not using this anymore with the whole pytest test suite.

Reply:
awesome was actually wondering about that.